app/src/main/java/com/celzero/bravedns/service/PersistentState.kt

```diff
@@ -208,62 +176,44 @@ class PersistentState(private val context: Context):SimpleKrate(context) {
         HomeScreenActivity.GlobalVariable.isBackgroundEnabled = backgroundEnabled
     }
 
-    fun setScreenLockData(isEnabled : Boolean) {
-        HomeScreenActivity.GlobalVariable.isScreenLocked = if(isEnabled) 1
-        else 0
-        isScreenOff = isEnabled
-    }
-
-    fun getScreenLockData(): Boolean {
-        return when (HomeScreenActivity.GlobalVariable.isScreenLocked) {
-            0 -> false
-            1 -> true
-            else -> isScreenOff
-        }
-    }
-
     //fixme replace the below logic once the DNS data is streamlined.
-    fun getConnectedDNS() : String{
-        if(connectedDNS.value.isNullOrEmpty()){
+    fun getConnectedDNS(): String {
+        if (connectedDNS.value.isNullOrEmpty()) {
             val dnsType = appMode?.getDNSType()
-            return if(dnsType == 1){
+            return if (dnsType == 1) {
                 val dohDetail: DoHEndpoint?
                 try {
                     dohDetail = appMode?.getDOHDetails()
                     dohDetail?.dohName!!
                 } catch (e: Exception) {
                     connectedDNSName
                 }
-            }else if(dnsType == 2){
-                if(appMode?.getDNSCryptServerCount() != null) {
+            } else if (dnsType == 2) {
+                if (appMode?.getDNSCryptServerCount() != null) {
                     val cryptDetails = appMode?.getDNSCryptServerCount()
-                     context.getString(R.string.configure_dns_crypt, cryptDetails.toString())
-                }else {
+                    context.getString(R.string.configure_dns_crypt, cryptDetails.toString())
+                } else {
                     context.getString(R.string.configure_dns_crypt, "0")
                 }
-            }else{
+            } else {
                 val proxyDetails = appMode?.getDNSProxyServerDetails()
                 proxyDetails?.proxyAppName!!
             }
-        }
-        else
-            return connectedDNS.value!!
+        } else return connectedDNS.value!!
     }
 
-    fun setConnectedDNS(name : String) {
+    fun setConnectedDNS(name: String) {
         connectedDNS.postValue(name)
         connectedDNSName = name
     }
 
-    fun setOrbotModePersistence(mode : Int){
+    fun setOrbotModePersistence(mode: Int) {
         orbotModeConst = mode
         orbotMode = mode
     }
 
-    fun getOrbotModePersistence(): Int{
-        if(orbotModeConst == 0)
-            return orbotMode
-        else
-            return orbotModeConst
+    fun getOrbotModePersistence(): Int {
+        if (orbotModeConst == 0) return orbotMode
+        else return orbotModeConst
     }
 }
```

Comment:
new line at the end of the file missing.

Reply:
Added new line.